package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "hobloom-twitch-chat-control",
+  "version": "2.1.2",
+  "description": "Twitch chat bot for allowing users to control your HOBloom growroom automatin",
+  "main": "tent-control-bot.js",
+  "directories": {
+    "lib": "lib",
+    "test": "tests"
+  },
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/roshkins/hobloom-twitch-chat-control.git"
+  },
+  "keywords": [
+    "HOBloom",
+    "growroom",
+    "Twitch",
+    "chat",
+    "chat",
+    "bot"
+  ],
+  "author": "atarimaster001",
+  "license": "GPL-3.0",
+  "bugs": {
+    "url": "https://github.com/roshkins/hobloom-twitch-chat-control/issues"
```

Comment:
Note to self: Don't forget to make sure these links update

Reply:
Ooh, lemme edit that url